tests/app/conftest.py

```diff
@@ -1,10 +1,12 @@
 # pylint: disable=redefined-outer-name
 
 from datetime import datetime, timedelta, timezone
+from unittest.mock import MagicMock
```

Comment:
```suggestion
from mock import MagicMock
```
we don't want to use unittest libraries with pytest